### PR TITLE
Renames routes so they are all consistent

### PR DESCRIPTION
**Pull request by pipwilson**
The renames the routes to all use hyphens instead of a mix of hyphenated-names and camelCaseNames.

It also renames the AboutPopController to AboutPersonController.

### CODE REVIEW

File: server/routes/about-person/AboutPersonController.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { NextFunction, Request, Response } from 'express'
 import locale from './locale.json'
 import { formatAssessmentData } from '../../utils/utils'
 
-export default class AboutPopController {
+export default class AboutPersonController {
```
Comment (alexw-moj) [inline server/routes/about-person/AboutPersonController.ts:5]: Url is just `/about` now, so just `AboutController` and `about` folder?

File: server/routes/about-person/routes.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import { Router } from 'express'
-import AboutPopController from './AboutPopController'
+import AboutPersonController from './AboutPersonController'
 import URLs from '../URLs'
 
 export default function setupAboutPopRoutes(router: Router) {
-  const controller = new AboutPopController()
+  const controller = new AboutPersonController()
 
   router.get(URLs.ABOUT_POP, controller.get)
```
Comment (alexw-moj) [inline server/routes/about-person/routes.ts]: `_POP`. Other function and variable names also use pop terminology too.

File: server/routes/about-person/AboutPersonController.ts (hunk 1 above)
Comment (pipwilson) [reply server/routes/about-person/AboutPersonController.ts:5]: I thought about that, but I thought this was clearer, similar to how we've left Plan as PlanOverview so there's a bit more of a clue to the humans on what the controller does.

File: server/routes/about-person/routes.ts (hunk 2 above)
Comment (pipwilson) [reply server/routes/about-person/routes.ts]: My main goal was getting the file structure consistent. I'll rename ABOUT_POP though and have a look through the other `pop`s.

**Comment by pipwilson** [comment]
Converted to draft until some of the other changes have come in - this change is not a priority right now.

**Pull request closed without merging**